Resources/config/listener.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    cachetool_cache_clearer:
+        class: CacheTool\Bundle\Listener\CacheClearerListener
+        arguments:
+            - @cachetool.cache_clearer.collection
```

Comment:
I dont think it is necessary to have a collection, just pass `cachetool` into this and call `$cachetool->opcache_reset()` and `$cacheool->apc_clear_cache('system')` and wrap each in a `try ... catch` block as an `InvalidArgumentException` will be thrown if the function wasn't loaded.
It would greatly simplify the implementation.